src/styles/variables/theme.scss

```
@@ -0,0 +1,15 @@
+// Переменные темы, которые можно будет изменять
+
+:root {
+  --theme-color-white: #fff;
+  --theme-color-basic: #000;
+  --theme-color-basic-middle: #696969;
+  --theme-color-basic-light: #EAEAEA;
+  --theme-color-accent: #399C82;
+  --theme-color-accent-middle: #BADCD3;
+  --theme-color-accent-light: #E2F2EE;
+  --theme-color-warning: #F3627B;
+  --theme-color-warning-middle: #FBC8D1;
+  --theme-color-warning-light: #FEEFF2;
+  --theme-color-background: #fff;
```

❗️
```suggestion
  --theme-color-warning-light: #FEEFF2;
  --theme-color-background: #FFF;
```

Или наоборот, но не оба кейса разом)

---

да нужен линтер)